bbp/default.nix

```diff
@@ -532,6 +532,13 @@ let
             branchName = "bare";
         };
 
+        neurodamus-syntool = callPackage ./hpc/neurodamus {
```

Comment:
For completeness, shouldn't we name this either `neurodamus-synapsetool` or `neurodamus-syn2`, the former to indicate packages used, the latter to state the file format support?

Reply:
I just didn't want it to become too verbose, so I went for a shorthand. syn2 is not a good candidate since in the future this version will be able to read other formats.
If you think the shorthand is confusing I can of course change it.

Reply:
It's already a long name, 4 more characters wouldn't hurt ("apse")? If more formats are added, is `syntool` not a misleading indicator, too, with `syn` in its name?

I guess that in the end, we might want to have that parameter always active and this version will disappear/be united with plain `neurodamus`, in which case I'm OK with the "temporary" name.